solutions/banking/variables.tf

```diff
@@ -4,11 +4,27 @@ variable "ibmcloud_api_key" {
   sensitive   = true
 }
 
+variable "prefix" {
```

Comment:
just a warning that we found consumes want full control over resource naming - if you don't really need a prefix, I would not add one here